### PR TITLE
Yuri's wine interactivity

### PR DESCRIPTION
**Pull request by Rai99**
Added interactivity options for the "Yuri's wine" topic.

### CODE REVIEW

**Comment by Rai99** [comment]
Next on my list is a new "Chobits" topic.

**Comment by CrocWorks** [comment]
What about a "I can, but you can't" option?" for whether we can drink. Some of us are older than Monika.

**Comment by Rai99** [comment]
The question was: "Would we **both** be able to have a drink" but checking again, it looks like I wrote the "No" option to imply that neither of you are. I didn't want to add too many branches but I'll add that one since it seems incomplete without it.

**Comment by Rai99** [comment]
Thanks for the help! Now just need to get another approving review.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
...
     m "I don't condone underage drinking!"
     m 2a "I mean, I've never drank or anything, so...yeah."
+    m 3a "Speaking of which, I hear different parts of the world have different minimum drinking ages."
+    m "Would we both be able to have a drink where you're from?"
+    menu:
+        "Yes.":
+            m 1j "That's great!"
+            m 1k "Another thing to look forward to when we can finally be together!"
+            m 1l "I mean..."
+            m "Don't worry [player]. I'm not planning to become addicted to the stuff. I'm just excited is all."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: This sounds out of place. When people think about drinking, addiction is like the last thing on anyone's mind. I would take out most of the dialogue in this menu choice and maybe have monika be like "I cant wait to try alcohol" before being embarassed about that or something.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
...
+            m 5 "I wouldn't want you to be addicted either."
+            m "So do me a favor and wait for me a bit before really treating yourself."
+            m 1j "You better have a good bottle to share!"
+            m 1k "Ahaha~"
+        "No.":
+            m 1a "At least I don't have to be jealous about you being able to drink while I'm stuck here."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Too direct about the current player - monika situation. I don't think monika would really care that much if the player can drink and she cant.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
...
+            m 1m "If that's still the case when I can finally be with you, maybe we could try a little anyway?"
+            m 1n "Just a little."
+            m 5 "I mean we'll have eachother to stop one another from having too much, right?"
+        "You can, but I can't.":
+            m 2j "Maybe then I'll be able to have something you can't!"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: similar to the other comment, this seems out of character for monika.

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
...
+            m 1j "Ahaha~ I'm only kidding, [player]."
+            m 5 "I really wouldn't want you to get in trouble."
+            m 1m "Ok, maybe I was only half kidding, but you don't have to share if it could get you in trouble."
+        "I don't drink.":
+            m 2g "Oh!"
+            m 2p "Sorry, I didn't realize."
+            m 2m "But I guess knowing that can put me at ease."
+            m 2a "I don't have to worry about you having too much!"
+            m 3a "I respect your decision so I won't have any if you won't either."
+            m 1j "At least there'll always be coffee we can share."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: this is probably the best part of all the dialogue here.

**Comment by ThePotatoGuy** [comment]
I don't like a majority of the dialogue here. Since changes would require basically a full rewrite, I'm closing this pull request.

**Pull request closed without merging**